db/schema.rb

```diff
@@ -95,17 +95,21 @@
   add_index "applications", ["suburb"], name: "index_applications_on_suburb", using: :btree
 
   create_table "authorities", force: true do |t|
-    t.string  "full_name",            limit: 200, null: false
-    t.string  "short_name",           limit: 100, null: false
+    t.string  "full_name",                    limit: 200,                 null: false
+    t.string  "short_name",                   limit: 100,                 null: false
     t.boolean "disabled"
-    t.string  "state",                limit: 20
+    t.string  "state",                        limit: 20
     t.string  "email"
     t.integer "population_2011"
     t.text    "last_scraper_run_log"
     t.string  "morph_name"
     t.boolean "write_to_councillors_enabled",             default: false, null: false
+    t.string  "lga_name15"
+    t.string  "superceded_by"
```

Comment:
Unfortunately that'll keep appearing due to me not dropping / recreating the db from scratch in a number of pull requests - only realised after the first one or two. Silly db/schema, why u not feature branch well.